Tools/TestWebKitAPI/Tests/WebKitCocoa/TestSOAuthorization.mm

```diff
@@ -250,28 +250,24 @@ @interface TestSOAuthorizationScriptMessageHandler : NSObject <WKScriptMessageHa
 
 @implementation TestSOAuthorizationScriptMessageHandler {
     RetainPtr<NSMutableArray> _messages;
+    RetainPtr<NSArray> _expectedMessages;
+}
+
+- (instancetype)initWithExpectation:(NSArray *)expectedMessages
+{
+    _messages = adoptNS([[NSMutableArray alloc] init]);
+    _expectedMessages = adoptNS(expectedMessages);
```

Comment:
(Nit - I don't think you mean to `adoptNS` here? Since `expectedMessages` is autoreleased at all call sites)